src/runcrate/convert.py

```diff
@@ -362,13 +358,18 @@ def add_tool(self, crate, workflow, cwl_tool):
             properties["@type"] = "SoftwareApplication"
         if hasattr(cwl_tool, "intent") and cwl_tool.intent:
             properties["featureList"] = cwl_tool.intent
+        if hasattr(cwl_tool, "requirements") and cwl_tool.requirements:
+            for req in cwl_tool.requirements:
+                if req.class_ == "ResourceRequirement":
+                    ramMin = req.ramMin
+                    if ramMin:
+                        properties["memoryRequirements"] = f"{int(ramMin)} MiB"
```

Comment:
@simleo Would you like a test for this?

Reply:
It would be nice, but I guess we need an RO bundle in the test data that uses "requirements" instead of "hints" in order to test it, don't we?

Reply:
Yep. I can copy & modify an existing bundle, or we can leave it be.